admin/app/helpers/solidus_admin/importmap_helper.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module SolidusAdmin::ImportmapHelper
+  def javascript_solidus_admin_importmap_tags(entry_point = "solidus_admin/application", shim: false, importmap: SolidusAdmin.importmap)
```

Comment:
Sorry, @elia, any reason why we defaulted `shim` to `false` in our version? At first, glance, adding the `shim` looks nicer for greater compatibility.

Reply:
Yes of course! It was set to false for two reasons:
1. we are in a controlled environment in which is reasonable to expect evergreen browsers that will support import maps out of the box https://caniuse.com/import-maps
2. this allowed to remove an extra import in the component stimulus loader code that is still visible in the original file https://github.com/hotwired/stimulus-rails/blob/498902a3687d32bd6589f7d3c7e7d72eabd822c0/app/assets/javascripts/stimulus-loading.js#L1-L2